tests/cli/mirroring_smoke_tests/e2e/fluvio-core.bats

```diff
@@ -0,0 +1,189 @@
+    CURRENT_DATE=$(date +%Y-%m)
+    export CURRENT_DATE
+
+    REMOTE_PROFILE=local
+    export REMOTE_PROFILE
+    debug_msg "Remote profile: $REMOTE_PROFILE"
+
+    REMOTE_NAME=edge1
+    export REMOTE_NAME
+    debug_msg "Remote name: $REMOTE_NAME"
+
+    REMOTE_PROFILE_2=local2
+    export REMOTE_PROFILE_2
+    debug_msg "Remote profile: $REMOTE_PROFILE_2"
+
+    REMOTE_NAME_2=edge2
+    export REMOTE_NAME_2
+    debug_msg "Remote name: $REMOTE_NAME_2"
+
+    MESSAGE="$(random_string 7)"
+    export MESSAGE
+    debug_msg "$MESSAGE"
+
+    TOPIC_NAME=mirror-topic
+    export TOPIC_NAME
+    debug_msg "Topic name: $TOPIC_NAME"
+
+    HOME_PROFILE=$($FLUVIO_BIN profile)
+    export HOME_PROFILE
+    debug_msg "Home profile: $HOME_PROFILE"
+
+    HOME_NAME="home"
+    export HOME_NAME
+    debug_msg "Home name: $HOME_NAME"\
+
+    K8S_DEFAULT_PORT=":30003"
+    export K8S_DEFAULT_PORT
+    debug_msg "K8S default port: $K8S_DEFAULT_PORT"
+}
+
+@test "Can register first local remote cluster" {
+    run timeout 15s "$FLUVIO_BIN" remote register "$REMOTE_NAME"
+
+    assert_output "remote cluster \"$REMOTE_NAME\" was registered"
+    assert_success
+}
+
+@test "Can register second local remote cluster" {
+    run timeout 15s "$FLUVIO_BIN" remote register "$REMOTE_NAME_2"
+
+    assert_output "remote cluster \"$REMOTE_NAME_2\" was registered"
+    assert_success
+}
+
+@test "Export home to remote 1" {
+    run timeout 15s "$FLUVIO_BIN" remote export "$REMOTE_NAME" --file remote.json
+    assert_output ""
+    assert_success
+
+    run jq -r .home.id remote.json 
+    assert_output "$HOME_NAME"
+    assert_success
+
+    run jq -r .home.remoteId remote.json 
+    assert_output "$REMOTE_NAME"
+    assert_success
+
+    run jq -r .home.publicEndpoint remote.json 
+    assert_output --partial "$K8S_DEFAULT_PORT"
+    assert_success
+}
+
+@test "Export home to remote 2" {
+    run timeout 15s "$FLUVIO_BIN" remote export "$REMOTE_NAME_2" --file remote2.json
+    assert_output ""
+    assert_success
+
+    run jq -r .home.id remote2.json 
+    assert_output "$HOME_NAME"
+    assert_success
+
+    run jq -r .home.remoteId remote2.json 
+    assert_output "$REMOTE_NAME_2"
+    assert_success
+
+    run jq -r .home.publicEndpoint remote2.json 
+    assert_output --partial "$K8S_DEFAULT_PORT"
+    assert_success
+}
+
+@test "Can create a mirror topic with the remote clusters" {
+    echo "[\"$REMOTE_NAME\",\"$REMOTE_NAME_2\"]" > remotes_devices.json
+    run timeout 15s "$FLUVIO_BIN" topic create "$TOPIC_NAME" --mirror-apply remotes_devices.json
+
+    assert_output "topic \"$TOPIC_NAME\" created"
+    assert_success
+}
+
+@test "Can switch to remote cluster 1" {
+    run timeout 15s "$FLUVIO_BIN" profile switch "$REMOTE_PROFILE"
+    assert_output ""
+    assert_success
+}
+
+@test "Can't produce message to mirror topic from remote 1 before connect to home" {
+    sleep 5
+    run bash -c 'echo 1 | timeout 15s "$FLUVIO_BIN" produce "$TOPIC_NAME"'
```

Comment:
this should work.  initial json spec should contain mirror topic config

Reply:
Really? The main idea of SC-to-SC is not sync the topics?

Reply:
Yes but initial json should contain mirror topics.   This means remote cluster should be able to produce to local topic without connecting to home first.

Reply:
I thought that it was a limitation of using read_only mode. Make sense. It should be covered on this PR?

Reply:
I put a task in the roadmap to fix it https://github.com/infinyon/roadmap/issues/277

Reply:
Ok. this seems at least cover most of core use cases